app/controllers/application_controller.rb

```diff
@@ -200,7 +200,7 @@ def find_version
       end
       raise ActiveRecord::RecordNotFound if @tags.empty? && params[:number].present?
     else
-      @versions = @project.versions.sort.first(10)
+      @versions = @project.versions.order(number: :desc).limit(10)
```

Comment:
Would it make more sense to use `published_at` for this? It looks like the sorting for `Version` tends to use that first before defaulting to a string comparison on the version number if I'm reading the sort code correctly. That also might prevent weird versions showing up at the top of the list like the occasional `v99999.9999.9999` that show up every once in a while.

https://github.com/librariesio/libraries.io/blob/71da9e378da59ddd69dbaf7132bf8e934421cb8c/app/models/version.rb#L145-L151

Reply:
Changed to sort by `published_at` and added an on `versions.published_at`